README.md

```diff
@@ -1,2 +1,6 @@
-# CR-Trials-Mobile-Menu
-This is an exercise for the Microverse Code Reviewer Trials.
+# curriculum-cr-training-mobile-menu
+
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
```

Comment:
- [ ] README file should be descriptive. please use the  [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse